repo/packages/H/hue/0/config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "type": "object",
+  "properties": {
+    "hdfs": {
+      "description": "Hadoop specific configuration properties",
+      "type": "object",
+      "properties": {
+        "scheduler-uri": {
```

Comment:
nit: In description, it should say (must being with 'hdfs-mesos') instead of "description": "HDFS package artifact tarball (must begin with 'hdfs')"